apps/alerts/lib/cache/store.ex

```diff
@@ -133,13 +140,30 @@ defmodule Alerts.Cache.Store do
         }
       end)
 
+    stop_inserts =
+      Enum.flat_map(alerts, fn alert ->
+        non_nil_stops =
+          Enum.filter(alert.informed_entity.stop, fn stop_id ->
+            stop_id != nil
+          end)
```

Comment:
Do alerts really have `informed_entity.stop` values of mixed nil and non-nil IDs? Seems a bit odd to me.

Reply:
That is what I noticed in testing this.